app/assets/stylesheets/report.scss

```diff
@@ -2,26 +2,39 @@
   background: #fff;
   border-radius: 2px;
   box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24);
-  display: block;
-  float: left;
-  margin: 1rem;
+  // display: block;
+  // float: left;
+  margin-top: 2rem;
+  margin-bottom: 2rem;
   min-height: 300px;
-  position: relative;
+  // position: relative;
   transition: all 0.2s ease-in-out;
-  width: 100%;
-  
+  // width: 100%;
+  padding: 15px 15px 15px 15px;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `15px` instead of `15px 15px 15px 15px`